starters/nestjs-bullmq-postgresql/src/main.ts

```diff
@@ -1,8 +1,16 @@
 import { NestFactory } from '@nestjs/core';
 import { AppModule } from './app.module';
+import 'dotenv/config';
 
 async function bootstrap() {
 	const app = await NestFactory.create(AppModule);
+	app.enableCors({
+		origin: process.env.origins_urls ? process.env.origins_urls.split(',') : false,
+		methods: 'GET,HEAD,PUT,PATCH,POST,DELETE',
+		preflightContinue: false,
+		optionsSuccessStatus: 204,
```

Comment:
I think those 3 options `methods`, `preflightContinue`, and `optionsSuccessStatus` are basically default values so no need to specify them